packages/web-client/src/helpers/share/functions.ts

```diff
@@ -151,9 +150,7 @@ export function buildIncomingShareResource({
     sharePermissions,
     outgoing: false,
     canRename: () => driveItem['@client.synchronize'],
-    canDownload: () =>
-      sharePermissions.includes(GraphSharePermission.readBasic) &&
-      !shareRoles.some((shareRole) => shareRole.id === GraphShareRoleIdMap.SecureViewer),
+    canDownload: () => sharePermissions.includes(GraphSharePermission.readContent),
```

Comment:
So do I get it right that `readContent` permission action is not given with a secure view share? Which permission action does the secure view share have for secure-viewing content?

Reply:
Okay so the permissions of a secure view share are these:

```
* libre.graph/driveItem/path/read
* libre.graph/driveItem/children/read
* libre.graph/driveItem/basic/read
```